src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -1,6 +1,8 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+using System.Text.Json;
+using System.Text.Unicode;
```

Comment:
I guess `System.Text.Unicode` can be removed? I think it was only used in the first attempt where we used `UnicodeRanges.All` for the encoder